example/lib/peripheral_details/peripheral_detail_page.dart

```diff
@@ -146,13 +146,8 @@ class _PeripheralDetailPageState extends State<PeripheralDetailPage> {
     }
 
     try {
-      await selectedCharacteristic.write(
-        value,
-        withResponse: _hasSelectedCharacteristicProperty(
-          [CharacteristicProperty.writeWithoutResponse],
-        ),
-      );
-      _addLog('Write', value);
+      await selectedCharacteristic.write(value, withResponse: withResponse);
+      _addLog('Write${!withResponse ? "WithoutResponse" : ""}', value);
```

Comment:
```suggestion
      _addLog('Write${withResponse ? "" : "WithoutResponse"}', value);
```
We can avoid the negation